deploy/docs/Installation_with_Helm.md

```diff
@@ -0,0 +1,98 @@
+- [How to install our Prometheus side by side with your existing Prometheus](https://github.com/SumoLogic/sumologic-kubernetes-collection/deploy/docs/SideBySidePrometheus.md)
+
+To install the chart, first add the `sumologic` private repo:
+
+```bash
+helm repo add sumologic https://sumologic.github.io/sumologic-kubernetes-collection
+```
+
+Install the chart with release name `collection` and namespace `sumologic`
+
+```bash
+helm install sumologic/sumologic --name collection --namespace sumologic --set sumologic.endpoint=<SUMO_ENDPOINT> --set sumologic.accessId=<SUMO_ACCESS_ID> --set sumologic.accessKey=<SUMO_ACCESS_KEY>
+```
+
+If you get `Error: customresourcedefinitions.apiextensions.k8s.io "alertmanagers.monitoring.coreos.com" already exists`, run
+
+```bash
+helm install sumologic/sumologic --name collection --namespace sumologic --set sumologic.endpoint=<SUMO_ENDPOINT> --set sumologic.accessId=<SUMO_ACCESS_ID> --set sumologic.accessKey=<SUMO_ACCESS_KEY> --no-crd-hook
+```
+
+To customize your configuration, download the values.yaml file by running
+
+```bash
+curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/master/deploy/helm/sumologic/values.yaml
+```
+
+NOTE: If you need to install the chart with a different release name or namespace you will need to override some configuration fields for both Prometheus and fluent-bit. We recommend using an override file due to the number of fields that need to be overridden. In the following command, replace the `<RELEASE-NAME>` and `<NAMESPACE>` variables with your values and then run it to download the override file with your replaced values:
+
+```bash
+curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/master/deploy/helm/sumologic/values.yaml | \
+sed 's/\-sumologic.sumologic'"/-sumologic.<NAMESPACE>/g" | \
+sed 's/\- sumologic'"/- <NAMESPACE>/g" | \
+sed 's/\collection'"/<RELEASE-NAME>/g" > values.yaml
+```
+
+For example, if your release name is `my-release` and namespace is `my-namespace`:
+```bash
+curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/master/deploy/helm/sumologic/values.yaml | \
+sed 's/\-sumologic.sumologic'"/-sumologic.my-namespace/g" | \
+sed 's/\collection'"/my-release/g" > values.yaml
+```
+
+Make any changes to the `values.yaml` file as needed, and run the following to install the chart with the override file.
+
+```bash
+helm install sumologic/sumologic --name my-release --namespace my-namespace -f values.yaml --set sumologic.endpoint=<SUMO_ENDPOINT> --set sumologic.accessId=<SUMO_ACCESS_ID> --set sumologic.accessKey=<SUMO_ACCESS_KEY> 
+```
+
+If you would like to use a different cluster name than the default `kubernetes`, add this to the `helm install` command:
+```
+--set prometheus-operator.prometheus.prometheusSpec.externalLabels.cluster="<my-cluster-name>" --set sumologic.clusterName="<my-cluster-name>"
+```
+
+## Uninstalling the Chart
```

Comment:
Since our process is not yet idempotent we should doc deleting the secret and collector